src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -89,6 +94,24 @@ public function getBaseHref(): ?string
         return $this->baseHref;
     }
 
+    /**
+     * Returns usage of masterminds/html5 or not.
+     */
+    public function isHtml5Parser(): bool
+    {
+        return $this->isHtml5Parser;
+    }
+
+    /**
+     * Set usage of masterminds/html5 or not.
+     */
+    public function setIsHtml5Parser(bool $isHtml5Parser = true): static
```

Comment:
This should be set though a constructor argument instead. Otherwise, it is incompatible with cases passing the content in the constructor as a string.